src/aurora.ts

```diff
@@ -459,7 +473,10 @@ export class Aurora extends Construct {
       );
 
       if (!props.skipAddRotationMultiUser) {
-        const rotation = this.cluster.addRotationMultiUser(user.pascal, { secret });
+        const rotation = this.cluster.addRotationMultiUser(user.pascal, {
+          secret,
+          ...passwordRotationOptions,
```

Comment:
We should guard against passwordRotationOptions getting expanding and having a `secret` entry.
```suggestion
          ...passwordRotationOptions,
          secret,
```

Reply:
Better still, just pass a `CommonRotationUserOptions` type